src/main/app/claims/claimStoreClient.ts

```diff
@@ -1,4 +1,4 @@
-import { request } from 'client/request'
+import { retryingRequest } from 'client/request'
```

Comment:
Didn't we agree to use retrying request by default in all places?

Reply:
As agreed it will be first applied to claim store calls only and then to the rest.

Reply:
I am happy with using `retryingRequest` for now to emphasise different behaviour in claim-store client. Later when we apply retrying request to all other places we should rename it back to be `request`